editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -1201,7 +1210,7 @@ export const NavigatorRowLabel = React.memo((props: NavigatorRowLabelProps) => {
         gap: 5,
         borderRadius: 20,
         height: 22,
-        paddingLeft: 5,
+        paddingLeft: props.hideExpandableIndicator ? 0 : 5,
```

Comment:
won't this push conditional clauses to the left? or is that desired?

Reply:
oh right, good point. fixed in https://github.com/concrete-utopia/utopia/pull/5768/commits/247cd34af1f56907dc12eb854ee3dcb5dedf82bd